.github/workflows/build.yml

```
@@ -68,3 +68,15 @@ jobs:
         run: |
           lean --recursive --export=mathlib.txt src/
           leanchecker mathlib.txt
+      - name: Set up Python
```

```suggestion

      - name: Set up Python
```

That is, just add an empty line above.